lib/umwelt/client/fetcher.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Umwelt.Client.Fetcher do
+  @moduledoc "Code fetcher task"
+
+  use Task
+  require Logger
+
+  alias Umwelt.Client
+
+  def start_link(module) do
+    Task.start_link(__MODULE__, :run, [module])
+  end
+
+  def run(module) do
+    case Client.Request.fetch_code(module) do
+      {:ok, code} ->
+        Logger.debug("Success fetch #{module.name}")
+        send(Client.Clone, {:fetched, %{name: module.name, code: code}})
```

Comment:
IS there any reason you prefer `send/2`/`handle_info/2` over `GenServer.cast/2`/`handle_cast/2`?

Reply:
It was inspired by [this thread](https://elixirforum.com/t/genserver-handle-cast-vs-handle-info/44636). Looks reasonable but not sure.